environments/01-network/test.tfvars

```diff
@@ -0,0 +1,38 @@
+enable_debug = "true"
+
+network_address_space                  = "10.48.64.0/18"
+aks_00_subnet_cidr_blocks              = "10.48.16.0/20"
+aks_01_subnet_cidr_blocks              = "10.48.32.0/20"
+iaas_subnet_cidr_blocks                = "10.48.1.0/24"
+application_gateway_subnet_cidr_blocks = "10.48.0.128/25"
+
```

Comment:
The `network_address_space` does not match the proposed used subnets

Reply:
corrected.